README.md

```diff
@@ -72,7 +84,7 @@ It is recommended that you use environment variables to configure the Pi-hole do
 
 ### Web interface password
 
-To set a specific password for the web interface, use the environment variable `FTLCONF_webserver_api_password`. If this variable is not detected, and you have not already set one via `pihole setpassword` in the container, then a random password will be assigned on startup, this will be printed to the log. Run `docker logs pihole | grep random` to find it.
+To set a specific password for the web interface, use the environment variable `FTLCONF_webserver_api_password`. If this variable is not detected, and you have not already set one via `pihole setpassword` / `pihole-FTL --config webserver.api.password` inside the container, then a random password will be assigned on startup, this will be printed to the log. Run `docker logs pihole | grep random password` to find it.
```

Comment:
Changing the password is not possible when the `FTLCONF_webserver_api_password` variable is set (and any other option set by ENV VAR), but there is no explanation on the README about this ENV VARs behavior.

Reply:
Ok, I think we can mention that somewhere else on the Readme, it's a bit of a mouthful for this section

Reply:
I agree. I just added the comment here because it was easier, but I think this could be in the variables section.